cves/kernel/CVE-2018-20961.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: Some USB gadgets have multiple 'modes' devices that can switch between modes 
+  and possibly cause a double-free flaw. Subsequently the USB gadget Midi driver 
+  in the Linux kernel created a double-free when handling certain errors. 
```

Comment:
Good, but if nothing else maybe explain what a double free means/why it crashes and how it crashes